draft-ietf-keytrans-architecture.md

```diff
@@ -400,6 +400,94 @@ Alice                  Transparency Log                  Manager
 Transparency Log to the Manager. Rejected requests are blocked." }
 
 
+# Combining Logs
+
+There are many cases where it makes sense to operate multiple cooperating log
+instances, for example:
+
+- A service provider may decide that it's prudent to rotate its cryptographic
+  keys, or migrate to a new deployment mode. They can do this by creating a new
+  log instance with new cryptographic keys, operating under a new
+  deployment mode if desired, and migrating their data from the old log to the
+  new log while users are able to query both.
+- A service provider may choose to operate multiple logs to improve their
+  ability to scale or provide higher availability.
+- A federated system may allow each participant in the federation to operate
+  their own log for their own users.
+
+Client implementations should generally be prepared to interact with multiple
+logs simultaneously. In particular, clients SHOULD namespace any configuration
```

Comment:
What happens if conflicts exist? Why not MUST?

Reply:
"SHOULD" because I think supporting multiple logs should be the default decision, unless an implementer really really knows they're only ever going to talk to one log at a time